client.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```

Comment:
Can we make debugging conditional?

```
if [[ -v DEBUG ]];then 
    set -x;
fi
```

Reply:
Akash, I always use `#!/bin/bash -x` on my scripts because I want to know what is going on, and I also want the users to know what is going on. I think it is a useful output, although it may seem a bit ugly in the beginning (until you get used to it). So, I don't know what would be the benefit of turning off debugging.